### PR TITLE
arpack: single-line body ifs matching the Ruby Style Guide

### PR DESCRIPTION
**Pull request by gabrielrussoc**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

A simple change to match the Ruby Style Guide. I'm getting started on contributing :)

### CODE REVIEW

File: Formula/arpack.rb [hunk 1]
```diff
@@ -38,9 +36,7 @@ def install
     pkgshare.install "TESTS/testA.mtx", "TESTS/dnsimp.f",
                      "TESTS/mmio.f", "TESTS/debug.h"
 
-    if build.with? "mpi"
-      (libexec/"bin").install (buildpath/"PARPACK/EXAMPLES/MPI").children
-    end
+    (libexec/"bin").install (buildpath/"PARPACK/EXAMPLES/MPI").children if build.with? "mpi"
```
Comment (DomT4) [inline Formula/arpack.rb]: The one above is fine, but this one shoots over 80 characters quite a bit, which may be why it was turned into a block.

Comment (gabrielrussoc) [reply Formula/arpack.rb]: The problem with the block is the `brew audit` warning. I tend to like the one-liner because it has no warnings.

Comment (DomT4) [reply Formula/arpack.rb]: Hmm. IMO it shouldn't be encouraging situations where that advice causes the line length to shoot more than 5% over the ideal 80 characters width. CC @Ilovezfs for thoughts.

Comment (DomT4) [reply Formula/arpack.rb]: I mean we could cheat & do something like:
```ruby
dir = buildpath/"PARPACK/EXAMPLES/MPI"
(libexec/"bin").install dir.children if build.with? "mpi"
```
But that seems mildly silly.

Comment (ilovezfs) [reply Formula/arpack.rb]: Current versions of rubocop will automatically flag these and if the change causes the line to go passed the eightieth column, then rubocop will not flag them. So if this was already passing brew style then there's nothing that needs to be changed.

Comment (DomT4) [reply Formula/arpack.rb]: Ah, you're correct, and I need to learn how to read apparently. Sorry! :see_no_evil:

```
== Library/Taps/homebrew/homebrew-core/Formula/arpack.rb ==
C: 27:  5: Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.

1 file inspected, 1 offense detected
```

Comment (ilovezfs) [reply Formula/arpack.rb]: So if any of these changes cause the line to exceed the eightieth column then we should not make that particular change, and if any of the changes don't make the line exceed the eightieth column then we should make that particular change.

Comment (ilovezfs) [reply Formula/arpack.rb]: OK, now I'm not on my phone. So I can see we should make the change at line 27 but not make the change at line 39.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/arpack.rb (hunk 1 above)
Comment (gabrielrussoc) [reply Formula/arpack.rb]: Indeed. I'm putting it back inside a block.

**Comment by fxcoudert** [comment]
Thanks @gabrielrussoc for your contribution! We look forward to other pull requests in the future 😉

**Pull request merged**